code/addons/a11y/package.json

```diff
@@ -32,8 +32,12 @@
       "require": "./dist/index.js",
       "import": "./dist/index.mjs"
     },
+    "./preview": {
+      "types": "./dist/preview.d.ts",
+      "require": "./dist/preview.js",
```

Comment:
@JReinhold while talking to @ndelangen he mentioned that this change is not enough, given that `./dist/preview.js` contents is actually ESM. We might need to make extra changes to actually generate a CJS file